Documentation/faq.md

```diff
@@ -0,0 +1,17 @@
+## Frequently Asked Questions (FAQ)
+
+### Configuration
+
+#### What is the difference between `advertise-urls` and `listen-urls`?
+
+`listen-urls` specifies the addresses etcd server binds to. etcd server accepts incoming connections from the interfaces it binds to. To listen on a port for all interfaces, specify `0.0.0.0` as listen IP address.
```

Comment:
`listen-urls` specifies the local addresses etcd server binds to for accepting incoming connections. To listen on a port for all interfaces, specify `0.0.0.0` as the listen IP address.